lib/use_cases/administrator/get_auth_requests_for_username.rb

```diff
@@ -6,8 +6,16 @@ def initialize(authentication_logs_gateway:)
       end
 
       def execute(username:)
+        return [] if username_invalid?(username)
+
         @authentication_logs_gateway.search(username: username)
       end
+
+    private
+
+      def username_invalid?(username)
+        username.nil? || username.empty? || username.length != 6
```

Comment:
I mentioned in Tom's PR that it would be nice to extract the username validator out of a similar piece of code #dry